docs/source/overview.rst

```diff
@@ -1,72 +1,97 @@
 Overview
 ========
 
-openprocurement.auctions.dgf contains documentaion for Deposit Guarantee Fund auctions.
+This ``openprocurement.auctions.lease`` package documentation contains information for Users
+of the PROZORRO.SALE system and intends to discover all aspects of ``lease procedure``.
 
-There are two procedures:
- 
- * dgfOtherAssets - sale of the insolvent bank property.
- 
- * dgfFinancialAssets - sale of the creditor claim right.
 
+The following subjects can act as an Organizer:
+
+* `State Property Fund of Ukraine`_;
+
+* state-owned enterprises;
+
+* the authorities, that are authorized by the Supreme Council of the Autonomous Republic of Crimea;
+
+* enterprises, institutions and organizations.
+
+
+To perform state property lease specialized procedure is available:
+
+ * ``propertyLease`` - lease of the state property.
 
 Features
 --------
 
-* The only date Organizer has to provide is *Tender.auctionPeriod.startDate*, the rest will be calculated automatically.
-* Optionally Organizer can set *enquiryPeriod.endDate*.
-* If *enquiryPeriod.endDate* is not provided it will be calculated automatically.
-* Organizer can both increase and decrease `value.amount`, `guarantee.amount`, `minimalStep.amount`.
-* `tenderPeriod` must be at least 7 calendar days.
-* Organizer can edit procedure only during *rectificationPeriod*.
-* Organizer can add and edit documents only during *rectificationPeriod*.
-* As soon as the action is edited, the status of all of the submitted bids will be switched to `invalid`.
-* Procedure can be switched from *draft* status to *active.tendering*.
-* During *active.tendering* period participants can ask questions, submit proposals, and upload documents.
-* There is obligatory participant qualification (*Bid.selfQualified*) via guarantee payment.
-* The only currency (*Value.currency*) for this procedure is hryvnia (UAH).
-* The items within an auction are allowed to be from different CAV groups.
+* The only date Organizer has to provide is ``Auction.auctionPeriod.startDate`` - approximate date when the auction starts - the actual value will be calculated automatically considering the existing capacity.
+
+* The rest dates and periods will be calculated automatically based on ``Auction.auctionPeriod.startDate``.
+
+* Optionally an Organizer can set ``rectificationPeriod.endDate``, if not provided - it will be calculated automatically.
+
+* ``tenderPeriod`` duration must be at least 7 calendar days.
+
+* Optionally an Organizer can set ``tenderPeriod.endDate`` - if so, auction can't start earlier 3 days after.
```

Comment:
Organizer can optionally set ``tenderPeriod.endDate``. The date provided should match the difference (3 working days) between ``auctionPeriod.startDate`` & ``tenderPeriod.endDate`` needed.